client/src/pages/Resource/Resource.scss

```diff
@@ -0,0 +1,14 @@
+@use "../../styles/layout";
+
+table.resource-details {
```

Comment:
Do you mean why write `table.resource-details` instead of just `.resource-details`? The former ensures that if the (relatively generic) class name `resource-details` gets used elsewhere on something that's _not_ a `table` element, these styles won't get accidentally applied.